services/catalog/src/simcore_service_catalog/db/repositories/groups.py

```diff
@@ -39,7 +39,9 @@ async def get_user_gid_from_email(
     ) -> Optional[PositiveInt]:
         async with self.db_engine.connect() as conn:
             return await conn.scalar(
-                sa.select([users.c.primary_gid]).where(users.c.email == user_email)
+                sa.select([users.c.primary_gid]).where(
+                    users.c.email == user_email.lower()
```

Comment:
This is not what I would have expected. I would have changed the email at the point of ingress in the application. At the edge (on the ingress API route). 
Doing so gives the guarantee that the email string is the same during the entire request and not treated differently in some cases.
I would track where this is coming from and change it accordingly at that entry point.